src/runtime_src/core/tools/common/reports/ReportAiePartitions.cpp

```diff
@@ -128,7 +128,7 @@ writeReport(const xrt_core::device* /*_pDevice*/,
     return;
   }
 
-  _output << boost::str(boost::format("Total Column Utilization: %s\n") % _pt.get<std::string>("aie_partitions.total_col_utilization"));
```

Comment:
L:112 puts "total_col_utilization" into the ptree for the json output; I think we should change it to "total_col_occupancy" for consistency. That would then change this line to something like `_output << boost::str(boost::format("Total Column Occupancy: %s\n") % _pt.get<std::string>("aie_partitions.total_col_occupancy"));`